applications/ContactStructuralMechanicsApplication/custom_utilities/contact_utilities.h

```diff
@@ -145,13 +145,13 @@ class ContactUtilities
         const double& tolerance = std::numeric_limits<double>::epsilon();
 
         // Initialize normal vectors
-        const array_1d<double,3> zero_vect = ZeroVector(3);
+        const array_1d<double,3>& zero_vect = ZeroVector(3);
```

Comment:
this & is a memory error

Reply:
or better said i don't understand what the compiler will do (ZeroVector is an expression not an array)